workload/master_config.yaml

```diff
@@ -0,0 +1,43 @@
+s3api:
+  # TEST_ID: TEST-35748
+  object_size:
+    start: 0Kib
+    end: 100Kib
+  part_range: #Multipart Specific
+    start: 50
+    end: 100
+  min_runtime_to_pass: 2h
+  sessions_per_node: 1
+  range_read: 100bytes
+  tool: s3api
+  operation: bucket/object_range_read/object_fix_size/object_random_size/copy_object/multipart/multipart_partcopy
```

Comment:
Should the operation be specified using / ? Or should it be list.
 Also if someone missed adding operation in config, which of above operation will be preformed by default.

Reply:
That was for reference, Let me change, I am keeping default value as bucket, and keeping all other operations in comments for user reference